Kickstarter-iOS/Features/PledgeOverTime/Views/PledgePaymentPlanOptionView.swift

```diff
@@ -168,3 +267,40 @@ private func applySubtitleLabelStyle(_ label: UILabel) {
 private func applySelectionIndicatorImageViewStyle(_ imageView: UIImageView) {
   imageView.contentMode = .center
 }
+
+private func applyTermsOfUseStyle(_ button: UIButton) {
+  button.configuration = {
+    var config = UIButton.Configuration.borderless()
+    config.contentInsets = NSDirectionalEdgeInsets(top: 1.0, leading: 0, bottom: 1.0, trailing: 0)
```

Comment:
nit: we could throw these inset values, and any others if we have any, in a Constants enum. Similarly to `EstimatedShippingCheckoutView.swift`

Reply:
Got it, I’ll move these `contentInsets` values (and any similar ones like spacing things) to a Constants enum for consistency. Thanks for the suggestion!